src/resolvers/fossil.cr

```diff
@@ -212,10 +212,13 @@ module Shards
       end
     end
 
+    def available_tags : Array(String)
+      tags = capture("fossil tag list -R #{Process.quote(local_fossil_file)}")
```

Comment:
```suggestion
      capture("fossil tag list -R #{Process.quote(local_fossil_file)}")
```

Reply:
This PR is still a draft while we're trying to figure out the intended behaviour. Commenting on superficial issues is just noise in this stage.
Let's focus on the contents first.

Reply:
Good point, sorry for the noise.